includes/EnvoimoinscherModel.php

```diff
@@ -1632,4 +1856,193 @@ public static function getNameCategory($id_eca)
 
 		return DB::getInstance()->getValue($query);
 	}
+
+	public function orderWithKeyExists($order, $key)
+	{
+		return count(Db::getInstance()->ExecuteS('SELECT * FROM '._DB_PREFIX_.'emc_orders eo
+		 JOIN '._DB_PREFIX_.'orders o ON o.id_order = eo.'._DB_PREFIX_.'orders_id_order
+		 WHERE eo.'._DB_PREFIX_.'orders_id_order = '.$order.' AND eo.tracking_eor = "'.$key.'" ')) > 0;
+	}
+
+	public function handlePush()
+	{
+		$type 		= Tools::getValue('type');
+		$key			= Tools::getValue('key');
+		$order 		= (int)Tools::getValue('order');
+		$return 	= false;
+		$emc 			= new Envoimoinscher();
```

Comment:
Done in the next revision here and on the other instances in EnvoimoinscherModel and EnvoimoinscherOrder.